Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
 RUN pip install mozilla-django-oidc
 RUN pip install spam_patterns@git+https://github.com/CUAHSI/spam_patterns.git@0.0.4
 RUN pip install freezegun
+RUN pip install --upgrade hsmodels
```

Comment:
when we release this, I would like to pin the hsmodels version in a newly minted image of hs_docker_base

Reply:
Definitely. Are you planning on doing the same for the other pip installs in this file?

Reply:
Yes. And ideally some of the other security issues we have
https://github.com/hydroshare/hs_docker_base/security/dependabot
Let's leave this PR open and once I get the base image for you, we can edit this Dockerfile and then merge?

Reply:
Okay I added all the security updates as well as new pinned hsmodels in the base image